app/routes.js

```diff
@@ -33,7 +33,7 @@ router.post('/lead-inspector/upload-planning-document', function (req, res) {
 
   // This is the URL the users will be redirected to once the email
   // has been sent
-  res.redirect('12345?planningDocUploaded=true');
+  res.redirect('Confirmation');
```

Comment:
It's conventional to use lowercase for URLs.

Reply:
Changed